.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Details
+___Do not mention proprietary info or link to internal work items in this PR.___
+
+**What were the changes?**  
+_One sentence describing the work done._
+
+**Why were the changes made?**  
+_Explain the motivation behind the work. Provide any publicly-available historical context._
+
+**How was the outcome achieved?**  
+_Technical details behind the work. Explain any publicly-available hardware peculiarities._
+
+**Additional Documentation:**  
+_What else should the reviewer know?_
+
+**Test Cases:**
+- [ ] _List here_
+
+## Approval Checklist
+- [ ] Ensure above details are appropriately filled
+- [ ] Ensure relevant internal documentation has been updated
+- [ ] Verify appropriate developer testing was done
+- [ ] Verify relevant CI is passing, if necessary
+- [ ] Verify test coverage has been added, if necessary
+- [ ] Verify the CHANGELOG has been updated, if necessary
```

Comment:
I think we need some clarifications on when to update CHANGELOG

Reply:
Agreed. My directive so far is that:
- Any changes that impact the customer should be logged.
- Any changes that impact any other team that is dependent on us should be logged.

Do you have any suggestions to further refine that?

Reply:
The directive looks good to me. Can you add them to template?

Reply:
I've added those and reworded to make them relevant to external authors and readers.